CleanCode.Naming/Rect_Class.cs

```diff
@@ -5,40 +5,43 @@
 
 namespace CleanCode.Naming
 {
-    class Rect_Class
+    class RectangleClass
     {
         // First side
         private int _a;
 
         // Second side
-        private int b;
-        private DateTime CDATE;
+        private int _b;
 
-        public Rect_Class(int a, int B)
+        // Creation DateTime
+        private DateTime _creationDate;
+
+        public RectangleClass(int a, int b)
         {
             _a = a;
-            b = B;
+            _b = b;
 
-            CDATE = DateTime.Now;
+            _creationDate = DateTime.Now;
         }
 
-        public int field()
+        public int GetField()
```

Comment:
raczej countField(bo funkcja jednak liczy to pole, a nie po prostu zwraca). Poza tym w C# są te Properties, które zastępują takie proste gety